lua/hbac/telescope/make_finder.lua

```diff
@@ -1,30 +1,32 @@
 local finders = require("telescope.finders")
+local results_opts = require("hbac.telescope.results_opts")
 
 local M = {}
 
-M.make_finder = function()
-	local make_display = require("hbac.telescope.make_display")
-	local utils = require("hbac.utils")
+local function get_entries(opts)
 
-	local display = make_display.display
+	local display = require("hbac.telescope.make_display").display
+	local buflist = require("hbac.utils").get_listed_buffers()
+	local filtered_bufnrs = {}
+	filtered_bufnrs, M.default_selection_idx = results_opts.filter_bufnrs_by_opts(opts, buflist)
```

Comment:
can we declare `default_selection_idx` inside M on L4, so that it's more obvious that it exists? Give it `nil` as default value or smth?